src/develop/ui/patterns/web/navigation/wizard.md

```diff
@@ -3,153 +3,108 @@ tags: runtime-traditionalweb;
 summary: Wizard splits complex tasks and process into steps.
 ---
 
-# Wizard
+# Wizard UI Pattern
+You can use the Wizard UI pattern to split large complex tasks and processes into smaller manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
 
-Split complex tasks and process into steps.
+**How to Use the Wizard UI Pattern**
 
-Use the Wizard to present numbered steps or conditions that the end user needs to complete or reach a goal. Use it to split large tasks into more manageable steps. Wizards usually include explicit button navigation to move a step forward or backward. 
+1. In Service Studio, in the Toolbox, search for  `Wizard`. 
 
-**How to use**
+    The Wizard widget is displayed.
+    
+     ![](<images/wizard-image-23.png>)
+
+1. From the Toolbox, drag the Wizard widget onto your application's screen.
 
-1. Drag Wizard pattern into the preview.
+    By default, the Wizard widget contains three Wizard Item blocks. Each Wizard Item represents a step. You can add or delete Wizard Items as required.
 
     ![](<images/wizard-image-1.png>)
 
-1. Drag as many WizardItems in the Content placeholder as required. 
+**Create a Wizard with Navigation Buttons**
 
-1. Set the content in the placeholders.
+The following example demonstrates how you can create a three step Wizard with navigation buttons.
 
-1. Set the mandatory values.
+1. Rename each of the Wizard Items to the following: 
+    * Shopping Details
+    * Payment Details
+    * Review Order
 
-    ![](<images/wizard-image-2.png>)
+        ![](images/wizard-image-14.png)
 
-1. Publish and test.
+1. Create an Input Parameter with the following properties:
+    
+    ![](images/wizard-image-13.png)
+ 
+    This input paramenter controls the current Wizard step. 
+    Setting the **Default Value** to 1 ensures the Wizard always starts at step 1.
 
+1. From the Toolbox, drag two Button widgets onto your screen and set their names to **Previous** and **Next**.
 
-## Input Parameters
+1. Define the behaviour for each of the navigation buttons. 
 
-### Wizard
+      ![](images/wizard-image-15.png)
 
-| **Input Name** |  **Description** |  **Type** | **Mandatory** | **Default Value** |
-|---|---|---|---|---|
-| Orientation  |  Set the orientation. | Orientation Identifier | False | Entities.Orientation.Horizontal |
-| ExtendedClass  |  Add custom style classes to this Block. | Text | False | none |
+1. Create an expression that defines the status of each of the Wizard Items (current step, step already completed, step yet to be completed), depending on what stage the user is at.
```

Comment:
This step is not really clear @annettewilliamsoutsystems @MiguelDomingues . Create an expression where - on screen? Or edit the expression property in an existing widget? And what does that expression actually mean? Please elaborate.

Reply:
Also, if we're expecting people to actually use that expression, it should be copyable from somewhere, rather than existing solely in an image.

Reply:
Edited

1. Create an expression that defines the status of each of the Wizard Items (active, past, next). Select the first Wizard Item (step 1), and on the **Properties** tab, from the **Step** drop-down, select **Expression Editor**. 

1. In the expression editor, enter the following expression:

    ``` If(CurrentStep = 1, Entities.Step.Active,If(CurrentStep > 1, Entities.Step.Past, Entities.Step.Next)) ```

    ![](images/wizard-image-16.png)

    This expression has the following meaning:
    * If the user is on step 1, the step will show as the active step in the wizard
    *If the user is on a step greater than step 1, step 1 will show as a step that is in thh past (completed).
    * If the user is on a step less than 1, the step will show as a next step.

    Repeat this for all of the Wizard Items.

I realize that the last bullet point doesn't really make sense but it is the correct logic, right?